actions/member.js

```diff
@@ -54,9 +60,9 @@ class List extends Action {
           }
         } : {})
       },
+      order: sorting({ defaultValue: [['firstName', 'ASC'], ['lastName', 'ASC'], ['cardId', 'ASC']] }),
```

Comment:
maybe it's better to just accept `defaultValue`

```suggestion
      order: sorting([['firstName', 'ASC'], ['lastName', 'ASC'], ['cardId', 'ASC']]),
```

could also ease usage as we are not supporting different sorting so `defaultValue` could be `{columns, asc=true}`